linux/LinuxProcessTable.c

```diff
@@ -334,23 +334,23 @@ static bool LinuxProcessTable_readStatFile(LinuxProcess* lp, openat_arg_t procFd
    location += 2;
 
    /* (4) ppid  -  %d */
-   Process_setParent(process, fast_strtol_dec(&location, 0));
+   Process_setParent(process, (pid_t) fast_strtol_dec(&location, 0));
    location += 1;
 
    /* (5) pgrp  -  %d */
-   process->pgrp = fast_strtol_dec(&location, 0);
+   process->pgrp = (int) fast_strtol_dec(&location, 0);
```

Comment:
```suggestion
   process->pgrp = (int)fast_strtol_dec(&location, 0);
```

Reply:
I would say I am a little tired of these `fast_strtol_dec` calls with `(int)` downcasts. Can we introduce a `fast_atoi_dec` and call it a day? The function can be modelled after `fast_strtol_dec`:

```c
static int fast_atoi_dec(char** str, int maxlen) {
   long result = fast_strtol_dec(str, maxlen);
   assert(result <= INT_MAX);
   assert(result >= INT_MIN);
   return (int)result;
}
```